src/components/content-tab-snapshots.tsx

```diff
@@ -379,6 +387,8 @@ function AddDemoSiteWithProgress( {
 	selectedSite: SiteDetails;
 	className?: string;
 	tagline?: string;
+	isExpired?: boolean;
+	onClearExpiredSite?: () => void;
```

Comment:
These two props seem not to be included in the props list of `AddDemoSiteWithProgress` component so we could remove them.

Reply:
Removed in https://github.com/Automattic/studio/pull/218/commits/32f5d9fc3fd4f72b209e61ad703bc2d6a0b8b4f2

nice catch! I was moving the button to be rendered on the same level as `addDemoSiteWithProgress` and these were the leftovers ✅